frontend/src/pages/projects/notebook/useRouteForNotebook.ts

```diff
@@ -19,12 +21,17 @@ const useRouteForNotebook = (
         return;
       }
       if (notebookName && projectName) {
-        getRoute(notebookName, projectName)
-          .then((route) => {
+        // if not using service mesh fetch openshift route, otherwise get Istio Ingress Gateway route
+        const getRoutePromise = dashboardConfig.spec.dashboardConfig.disableServiceMesh
```

Comment:
use the utlity function `featureFlagEnabled` to avoid confusion here or either use a const as I mentioned above.

Reply:
this file is outdated - not sure why it popped up - I have used the utility function [here](https://github.com/opendatahub-io/odh-dashboard/blob/c10666359453e5ac902b263c26ea6bb3c06659ca/frontend/src/pages/projects/notebook/useRouteForNotebook.ts#L25).

Reply:
great, thanks!